mango-cli/src/main/webapp/js/utilities.js

```diff
@@ -26,6 +26,16 @@ function toggleAlignments(sample, selector) {
         $(selector).slideToggle( "fast" );
 }
 
+// Toggles alignment record contained in the selector for a given sample
+function toggleVariants() {
```

Comment:
I'll remove them, I think you did. But shouldn't we have toggle functionality for the variants? that should be in your PR, right?